source/documentation/at-rules/import.html.md.erb

```diff
@@ -531,6 +531,56 @@ nice API for users of the new module system.
   @forward "reset" as reset-*
 <% end %>
 
+#### Configuring Modules Through Imports
+
+<% impl_status dart: '1.24.0', libsass: false, ruby: false %>
+
+You can configure modules that are loaded through an `@import` simply by
```

Comment:
Try to avoid words like "just" and "simply"—they can feel patronizing for some readers.

Also, link "configure modules" to the section describing how default variables work. Users reading this may not have the context to understand exactly what kind of "configuration" you're referring to.